configs/default/FOXE-FOXEERF745AIO

```diff
@@ -0,0 +1,135 @@
+# Betaflight / STM32F745 (S745) 4.1.0 Sep 25 2020 / 01:23:16 (064ca9b75) MSP API: 1.42
+
+board_name FOXEER_F745_AIO
```

Comment:
This will not work as a target definition as the file name is incorrect. Please read and follow the instructions in https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md hire somebody who has the skills to do so.

Reply:
Fixed the file name, sorry for issue, I missed ".config"